syntax = "proto3";

message OrderMessage {
    int32 id = 1;                // ID of the order
    int32 product_id = 2;         // ID of the product being ordered
    int32 quantity = 3;           // Quantity of the product ordered
    float total_price = 4;        // Total price of the order
    string status = 5;            // Status of the order, e.g., "pending", "completed"
    string action = 6;            // Action, e.g., "created", "updated", "deleted"
}
